### DISCUSSION TITLE
Allow other CSS units than only pixels in next/image

### DISCUSSION THREAD

**Discussion by krystof-k** [vercel/next.js#18661] [Ideas]
# Feature request

## Is your feature request related to a problem? Please describe.

I'd like to use rem units for example when defining next/image size.

## Describe the solution you'd like

Ability to pass other CSS units (especially rems) when defining next/image size. I understand that there may be a need for pixels anyway, so it would be OK to define both pixels and other unit, which will be used in the wrapper as the width property.

**Comment by thenderson55** [comment]
Any update on this?
